src/main/java/coresearch/cvurl/io/request/RequestBuilder.java

```diff
@@ -102,24 +106,35 @@ public T timeout(Duration timeout) {
         return (T) this;
     }
 
+    @SuppressWarnings("unchecked")
+    public T compressedResponse() {
+        this.compressedResponse = true;
+        return (T) this;
+    }
+
     /**
      * Builds new {@link Request}.
      *
      * @return new {@link Request}
      */
     public Request build() {
-        return new Request(setUpHttpRequestBuilder().build(), httpClient, genericMapper);
+        return new Request(setUpHttpRequestBuilder().build(), httpClient, genericMapper, compressedResponse);
```

Comment:
compessedResponse looks a bit wrong to me. I would prefer something like acceptCompessed